ccswbs/gus
# Add Statistics (New) widget

# Summary of changes
- Add new statistics module.

## Frontend
- Add 4 statistics styles: Left Border, Gradient of Solid Colours, Solid Colours, and Light Blue
- Allow StatIcon to pick up the colour from the statistics style

## Backend
- Add Statistics Item paragraph type (contains content for each statistic)
- Add Statistics Widget (New) paragraph type to contain Statistics Items
- Make Statistics Widget available on Basic Page in the Widgets, Section and Story paragraph types
- Add taxonomy type Statistics Styles to hold the available stat styles

# Test Plan

**Multidev:** http://storystat-chug.pantheonsite.io

1. View the page at https://storystat-chug.pantheonsite.io/node/972/edit?destination=/admin/content for an example of statistics in use
2. Create your own statistics example if you wish
3. Check the paragraph permissions
4. Review the page at https://mmtest.gatsbyjs.io/statistic-test

# Caveat
- When releasing the code, we will need to recreate the taxonomy items for Statistics Styles on Live as part of the release
